package.json

```diff
@@ -210,7 +210,7 @@
     "node": "^10.13.0"
   },
   "build": {
-    "appId": "com.loki-project.messenger-desktop",
+    "appId": "org.getsession.session-desktop",
```

Comment:
uhhh i wouldn't do this here and instead move it to another PR. This is because the `appId` is used as a unique app identifier on mac and windows. If we change it, it might break auto-updating or give us some errors on install. We should test it and see if any of those things happen before changing it.

Reply:
Alright I'll redact